dashboard/src/components/AppView/AppView.test.tsx

```diff
@@ -143,6 +143,24 @@ describe("AppViewComponent", () => {
       const sockets: WebSocket[] = wrapper.state("sockets");
       expect(sockets.length).toEqual(0);
     });
+
+    // See https://github.com/kubeapps/kubeapps/issues/632
+    it("handles manifests with duplicate keys", () => {
```

Comment:
nitpick: `"handles manifests with duplicate keys"` sounds too generic to me, I mean, handles it how? By capturing the error? By rendering it?

It might make sense to say something like `it supports manifests with duplicated keys`